src/app/myworkflows/my-workflow/my-workflow.component.html

```diff
@@ -61,20 +60,20 @@
                       >
                         Refresh All
                       </button>
-                      <a
-                        *ngIf="entryType === EntryType.Service"
-                        mat-menu-item
-                        [href]="gitHubAppInstallationLink$ | async"
-                        target="_blank"
-                        rel="noopener noreferrer"
-                        >Manage on GitHub</a
-                      >
-                    </mat-menu>
-                    <button mat-icon-button [matMenuTriggerFor]="appMenu">
-                      <mat-icon>
-                        more_vert
-                      </mat-icon>
-                    </button>
+                      <ng-container *ngIf="entryType === EntryType.Service">
```

Comment:
This is sort of an out there use case, because I'm not sure if we actually have any users out there like this, but in theory you could have a logged in w/Google user who doesn't have a GitHub account, in which case this button should probably be disabled or hidden.

Reply:
That's actually pretty interesting.  The installation is not tied to the Dockstore's GitHub account at all. It's entirely possible for the user to link an old GitHub account on Dockstore, log out of that GitHub account, login with a new GitHub account, and then click on the "Manage on GitHub" link.  The next page will ask to install on Dockstore on this new GitHub account but it may not be entirely obvious to the user that their old GitHub account is linked and not the new one.

I...do not have a solution for this.

Reply:
@coverbeck technically, the GitHub app is independent from the user's GitHub login. i.e. a user could login with Google, but then manage the GitHub app without ever letting us know about their GitHub account